sphinx/themes/basic/static/searchtools.js

```diff
@@ -137,6 +137,18 @@ const _displayNextItem = (
   // search finished, update title and status message
   else _finishSearch(resultCount);
 };
+const _orderResultsByScoreThenName = (a, b) => {
```

Comment:
It would be good to add a comment on where this function is used, what it does and what would be the layout of a and b.